src/utils/device.js

```diff
@@ -79,18 +69,13 @@ function isIOS () {
 }
 module.exports.isIOS = isIOS;
 
-function isGearVR () {
-  return /SamsungBrowser.+Mobile VR/i.test(window.navigator.userAgent);
-}
-module.exports.isGearVR = isGearVR;
```

Comment:
similar to the deprecation warnings in `src/utils/index.js`, would it be useful to first deprecate this method with a `console.warn` before removing?

I hope folks are not calling these methods, but I have seen them in various A-Frame projects recently.

Reply:
There's a warning that offers alternative method https://github.com/aframevr/aframe/pull/4033/files#diff-8968c62593320351ced6a0ef776ef599R30
Not only a a deprecation. isGearVR was no longer correct. It was checking for Samsung Internet browser. Written when Oculus Go was not a thing. There's no longer a way to discriminate between OculusGo and GearVR and probably not needed. `isMobileVR` replaces prior functionality.

Reply:
ah, understood. thanks for explaining.